recipe/meta.yaml

```diff
@@ -63,15 +63,610 @@ outputs:
     test:
       commands:
         - exit 0
+  # TODO: Extract parquet. Clean up build/host/run requirements
+  - name: libarrow-all
+    script: build-arrow.sh   # [unix]
+    script: build-arrow.bat  # [win]
```

Comment:
`libarrow-all` should just be a meta-package that pulls together the various sub-packages, compare e.g. the [ray](https://github.com/conda-forge/ray-packages-feedstock/blob/main/recipe/meta.yaml) feedstock.

In order to do the global build only once, you need to shift the build & host requirements out to the global level (unindent the `requirements:` block by 4 spaces and put it before `outputs:`), and then rename `build-arrow.sh` --> `build.sh` and `build-arrow-bat` --> `bld.bat`. The latter are magic names that conda will invoke once before tackling any of the outputs.

Reply:
Thanks a lot for all your tips and help! I've done that :)